### ISSUE TITLE
Form element: Return key does not trigger submit event listener

### ISSUE THREAD

**Issue by zachlearnscode** [primefaces/primevue#3146]
### Describe the bug

Between versions 3.15 and 3.16, something appears to be breaking `submit` event listeners on plain form elements.

From ^3.16.0, forms in my development and production builds don't fire their `submit` event callbacks when pressing the return key to submit the form. Of course, clicking the submit button works, and the enter key on the numpad on a full-size keyboard works. However, the return key, aka enter key grouped with all the alpha keys, does nothing.

I noticed there were breaking changes for accessibility introduced in v3.16.0 for several input components, and I wonder if changes there are creating this issue. Unfortunately, I can't recreate the issue on codesandbox, though I have provided a minimal reproduction of how my code is written below.

What leads me to believe that this is a bug in Primevue is that by simply reverting back to v3.15.0 or lower, the return key (as well as the other methods for submitting a form) work as expected.

### Reproducer

https://codesandbox.io/s/agitated-bartik-k6ziu4?file=/src/App.vue

### PrimeVue version

^3.16.0

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

### Browser(s)

Chrome 106, Firefox 106

**Comment by bahadirsofuoglu** [comment]
Hi,
We need more details because we cannot reproduce the problem. Is there a problem with PrimeVue version 3.17.0 as well? I don't think our accessibility improvements have had any effect. But if there is a development that you think is causing a problem, you can share it. Thank you

**Comment by zachlearnscode** [comment]
Hello. Yes, the issue on my machine and in my builds happens from Primevue 3.16.0 and up.

I've tried again to recreate the issue locally, without consistent replication. The issue is still at play in my project, but it doesn't seem like there's much work to be done here. The root of the problem is probably buried in some other dependency. I tried to get to the bottom of that as well, without success.

Thanks for your reply.
